modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -28,8 +30,11 @@
 /**
  * Facebook Instant Article node wrapper class.  Builds up an InstantArticle
  * object using field formatters.
+ *
+ * Class DrupalInstantArticleDisplay
+ * @package Drupal\fb_instant_articles_display
  */
-class InstantArticleNodeWrapper {
+class DrupalInstantArticleDisplay {
```

Comment:
@simonengelhardt Ha yeah, I thought a similar thing. I agree it would be better to change that, and I actually meant to change this to `DisplayInstantArticle`. The reason is I'm also working on refactoring that with a new class in the base module that extends `Facebook\InstantArticles\Elements\InstantArticle`, and then this Display class will then extend the one in the base module.